app/src/main/java/com/github/se/eventradar/model/event/EventList.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.se.eventradar.model.event
+
+data class EventList(
```

Comment:
comment: I know this is how the bootcamp did this, but I'm not convinced that this is the best way to structure the event list. We can go with it for now but I'm just stating my opinion.

Reply:
Do you have suggestions for better ways?

Reply:
Not yet, but again it's just my opinion.